### PR TITLE
Added Unexpected Maker TinyS2 board definition

### PR DESCRIPTION
**Pull request by UnexpectedMaker**
Added first pass board definitions for my new TinyS2 board

### CODE REVIEW

File: ports/esp32s2/boards/unexpectedmaker_tinys2/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,24 @@
+USB_VID = 0x303A
+USB_PID = 0x8002
+USB_PRODUCT = "TinyS2S2"
```
Comment (ladyada) [inline ports/esp32s2/boards/unexpectedmaker_tinys2/mpconfigboard.mk:3]: do you mean TinyS2 ?

Comment (UnexpectedMaker) [reply ports/esp32s2/boards/unexpectedmaker_tinys2/mpconfigboard.mk]: Yeah, whoops, fixed! How embarrassing ;)

**Comment by tannewt** [comment]
From the CI:
> Run python3 -u ci_new_boards_check.py
> Boards missing in .github/workflows/build.yml:
> unexpectedmaker_tinys2

You may want to install `pre-commit` as well. It'll auto-fix the issues it finds. https://pre-commit.com/#installation

**Comment by UnexpectedMaker** [comment]
> From the CI:
> 
> > Run python3 -u ci_new_boards_check.py
> > Boards missing in .github/workflows/build.yml:
> > unexpectedmaker_tinys2
> 
> You may want to install `pre-commit` as well. It'll auto-fix the issues it finds. https://pre-commit.com/#installation

Ok, I'll have a read through that doc and try to find some time to get it all installed and setup later this week, thanks :)

**Comment by UnexpectedMaker** [comment]
@tannewt I ran  python3 -u ci_new_boards_check.py (after installing pyjama and sh) and it did it's thing, and I can see tinys2 in the build.yaml file, but nothing new to commit. How do I get this  up to git to make the CI happy ?

**Pull request merged**